di_website/templates/includes/partials/author.html

```diff
@@ -14,6 +14,9 @@
         {% endif %}
     </div>
     <div class="m-author__body">
+        {% if "True" in guest_author %}
```

Comment:
Like Javascript, a string is truthy ... so I'm sure simply writing this as `if guest_author` will work.

Reply:
Updated, indeed this was a shifty hack